src/components/RequestInfo.tsx

```diff
@@ -0,0 +1,317 @@
+import { FunctionComponent, ReactNode, useEffect, useRef, useState } from 'react'
+
+import classNames from 'classnames'
+import { Timer, ListChecks, Check } from 'lucide-react'
+
+import { captureCustomEventWithPageData } from '../lib/utils'
+
+import { ContentSection } from './ContentSection'
+import { HubSpotForm } from './HubSpotForm'
+import { Layout } from './Layout/Layout'
+import ReadCaseStudyLink from './ReadCaseStudyLink'
+
+import styles from '../styles/CustomHubspotForm.module.scss'
+
+interface RequestInfoProps {
+    meta: { title: string; description: string }
+    formId?: string
+}
+const RequestInfo: FunctionComponent<RequestInfoProps> = ({ meta, formId }) => {
+    const formContainerRef = useRef<HTMLDivElement>(null)
+    const [formSubmitted, setFormSubmitted] = useState(false)
+
+    useEffect(() => {
+        if (formSubmitted && formContainerRef.current) {
+            formContainerRef.current.scrollIntoView({ behavior: 'smooth', block: 'start' })
+        }
+    }, [formSubmitted])
+
+    return (
+        <Layout
+            meta={meta}
+            className="bg-gray-50"
+            heroAndHeaderClassName="!pt-0"
+            hero={
+                <ContentSection className="pt-[97px] lg:pt-28" parentClassName="pt-11 lg:pb-16 px-6 lg:px-20">
+                    <div ref={formContainerRef}>
+                        <div className="relative mx-auto flex w-full flex-col pt-0 lg:flex-row lg:gap-[57px]">
+                            <div className="flex w-full max-w-[956px] flex-1 flex-col gap-y-20 lg:pt-0 lg:pr-0">
+                                <div>
+                                    <h1 className="mb-10 max-w-[524px] !text-5xl font-semibold text-gray-700 lg:!text-6xl lg:tracking-tighter">
+                                        Bring code intelligence to your organization
+                                    </h1>
+                                    <HeaderListTab contents={HeaderListTabItems} />
+                                </div>
+                                <div className="hidden gap-20 lg:flex lg:flex-col">
+                                    <SocialProofSection iconItems={SocialProofIcons} />
+                                    <CaseStudySection />
+                                </div>
+                            </div>
+                            <div className="w-full py-16 lg:w-1/2 lg:max-w-lg lg:px-6 lg:py-12">
+                                <div className="sg-border-gradient-glow relative z-10 !rounded-3xl border border-[#E4E9F4] bg-white pl-0 pt-12 pr-0">
+                                    <div className="flex flex-col items-center px-6 lg:px-12">
+                                        <h2 className="mb-3 text-[32px] !leading-[38.4px] text-gray-700">
+                                            Request demo
+                                        </h2>
+                                        <div className="text-center text-lg text-[#5E6E8C]">
```

Comment:
Can we make sure this text color (and any other color on the page) is assigned their proper Tailwind value? This should be `Gray-400`